pkg/reconciler/v1alpha1/taskrun/entrypoint/entrypoint_test.go

```diff
@@ -181,6 +185,44 @@ func TestGetRemoteEntrypoint(t *testing.T) {
 	}
 }
 
+func TestGetImageDigest(t *testing.T) {
+	img := getImage(t, &v1.ConfigFile{
+		ContainerConfig: v1.Config{},
+	})
+	digestHash, _ := img.Digest()
+	expectedDigetsSha := digestHash.String()
+	expectedRepo := "image"
+	manifestPath := fmt.Sprintf("/v2/%s/manifests/latest", expectedRepo)
```

Comment:
might be nice to have a function (or functions!) to return `digestHash` and `expectedRepo` especially since this seems to be repeated b/w the two tests

Reply:
Added only function to get digest like Str, since expetedRepo doesn't necessarily should be same for tests, and add function just to return on string seems like small overhead to me;)

Reply:
kk makes sense! :D